app/src/main/java/com/github/orkest/shazam/domain/ShazamConstants.kt

```diff
@@ -0,0 +1,39 @@
+package com.github.orkest.shazam.domain
+
+import android.Manifest
+import android.app.Activity
+import android.content.Context
+import android.content.pm.PackageManager
+import androidx.core.app.ActivityCompat
+import androidx.core.content.ContextCompat
+
+class ShazamConstants {
+    companion object{
+        const val REQUEST_RECORD_AUDIO_PERMISSION = 200
+        const val RECORDING_SAMPLE_RATE: Int  = 48_000
```

Comment:
Your forgot to use this constant in the audio recording file, you used a magic number instead

Reply:
Hahah indeed you are right, I will change it right away!